### PR TITLE
Jbs merge source audio

### PR DESCRIPTION
**Pull request by jsarabia**
This change is [Reviewable](https://reviewable.io/reviews/bible-translation-tools/otter/49)

### CODE REVIEW

**Comment by jsarabia** [comment]
This is currently just for me to visualize where I'm at and what's been changed

File: jvm/recorderapp/src/main/kotlin/org/wycliffeassociates/otter/jvm/recorder/app/view/SourceAudioFragment.kt [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    private fun initializeAudioNode(): AudioPlayerNode {
+        var sourceFile: File? = null
+        var startFrame: Int? = null
+        var endFrame: Int? = null
+        if (scope is ParameterizedScope) {
+            val parameters = (scope as? ParameterizedScope)?.parameters
+
+            parameters?.let {
+                val sourceAudio: String? = parameters.named["chapter_audio"]
+                sourceFile = if (sourceAudio != null && File(sourceAudio).exists()) File(sourceAudio) else null
```
Comment (mXaln) [inline jvm/recorderapp/src/main/kotlin/org/wycliffeassociates/otter/jvm/recorder/app/view/SourceAudioFragment.kt]: you're instantiating File(sourceAudio) twice here, maybe assign it to variable?

File: jvm/controls/src/main/kotlin/org/wycliffeassociates/otter/jvm/controls/skins/AudioPlayerSkin.kt [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+import javafx.scene.control.SkinBase
+import javafx.scene.control.Slider
+import javafx.scene.layout.HBox
+import org.kordamp.ikonli.javafx.FontIcon
+import org.wycliffeassociates.otter.common.device.AudioPlayerEvent
+import org.wycliffeassociates.otter.jvm.controls.AudioPlayerNode
+import org.wycliffeassociates.otter.jvm.utils.onChangeAndDoNow
+import java.util.concurrent.TimeUnit
+
+class AudioPlayerSkin(private val player: AudioPlayerNode) : SkinBase<AudioPlayerNode>(player) {
```
Comment (mXaln) [inline jvm/controls/src/main/kotlin/org/wycliffeassociates/otter/jvm/controls/skins/AudioPlayerSkin.kt]: This skin and SourceAudioSkin are very similar. I would make it one file with one fxml where particular items are shown or hidden depending on whether it's a source audio or simple audio.

**Comment by mXaln** [comment]
___
*[jvm/controls/src/main/kotlin/org/wycliffeassociates/otter/jvm/controls/skins/AudioPlayerSkin.kt, line 20 at r8](https://reviewable.io/reviews/bible-translation-tools/otter/49#-M0saM3Y-H4itUjoVHUa-r8-20:-M2hxPtF1RtvSJzydNvx:b11q5l) ([raw file](https://github.com/bible-translation-tools/otter/blob/2dc92c29516840cefd8df190237ba5839fb5085d/jvm/controls/src/main/kotlin/org/wycliffeassociates/otter/jvm/controls/skins/AudioPlayerSkin.kt#L20)):*
<details><summary><i>Previously, jsarabia (Joe) wrote…</i></summary><blockquote>

These two classes will diverge more substantially in the future with features like blocking out the audio when recording. I don't necessarily like the idea of adding a lot of complexity of hiding or showing functionality to share one FXML file. I am however working on in another branch pulling out the audio controller functionality to control swapping pause/play and updating progress/seek to one class that can be reused in audio player related cards or controls.
</blockquote></details>

all right!

**Pull request merged**